Please test in-game with `test/minescript_test.py`. It's failing due a mismatch in the expected number of fields in the `WorldInfo` class:

https://github.com/maxuser0/minescript/blob/main/test/minescript_test.py#L671

```
[world_info_test] Traceback (most recent call last):
  File "/Users/maxuser/code/minescript-dev/minescript/fabric/runs/client/minescript/minescript_test.py", line 778, in <module>
    test()
  File "/Users/maxuser/code/minescript-dev/minescript/fabric/runs/client/minescript/minescript_test.py", line 671, in world_info_test
    expect_equal(len(info.__dict__), 9)
  File "/Users/maxuser/code/minescript-dev/minescript/fabric/runs/client/minescript/minescript_test.py", line 82, in expect_equal
    raise TestFailure(f"Failed equality: {a} != {b}")
TestFailure: Failed equality: 10 != 9
```

--- a/test/minescript_test.py
+++ b/test/minescript_test.py
@@ -668,7 +668,8 @@ def screen_name_test():
 @test
 def world_info_test():
   info = minescript.world_info()
-  expect_equal(len(info.__dict__), 9)
+  expect_equal(len(info.__dict__), 10)
+  expect_equal(info.dimension, "minecraft:overworld")
 
 
 @test

--- a/test/minescript_test.py
+++ b/test/minescript_test.py
@@ -669,7 +669,7 @@ def screen_name_test():
 def world_info_test():
   info = minescript.world_info()
   expect_equal(len(info.__dict__), 10)
-  expect_equal(info.dimension, "minecraft:overworld")
+  expect_startswith(info.dimension, "minecraft:")
 
 
 @test

